src/gui/ResolveConflictsDialog.qml

```diff
@@ -34,9 +34,9 @@ ApplicationWindow {
     LayoutMirroring.childrenInherit: true
 
     width: Style.minimumWidthResolveConflictsDialog
-    height: Style.minimumHeightResolveConflictsDialog
+    height: Math.min(contentItem.height + 20, Screen.height * 0.8)
     minimumWidth: Style.minimumWidthResolveConflictsDialog
-    minimumHeight: Style.minimumHeightResolveConflictsDialog
+    minimumHeight: 200
```

Comment:
Rather than use a magic number here I think it'd be better to change the original value if that is needed